spec/spec_helper.rb

```diff
@@ -19,6 +19,7 @@
 end
 
 require 'ruby_smb/smb2'
+require 'ruby_smb/smb1'
```

Comment:
These shouldn't be necessary. Put all requires in to the top-level file (`lib/ruby_smb.rb`) and then you can just `require 'ruby_smb'`